packages/headless/src/features/triggers/triggers-slice.test.ts

```diff
@@ -76,4 +77,47 @@ describe('trigger slice', () => {
 
     expect(finalState.redirectTo).toEqual('https://www.coveo.com');
   });
+
+  it('when a executeSearch fulfilled is received and the payload does not contain any TriggerQuery objects, it does not update `state.query`', () => {
+    const state = getTriggerInitialState();
+    const triggers = [
+      buildMockTriggerNotify({content: 'notification'}),
+      buildMockTriggerRedirect({content: 'redirect'}),
+    ];
+    const response = buildMockSearchResponse({
+      triggers,
+    });
+    const searchState = buildMockSearch({
+      response,
+    });
+
+    const action = executeSearch.fulfilled(
+      searchState,
+      '',
+      logSearchboxSubmit()
+    );
+    const finalState = triggerReducer(state, action);
+
+    expect(finalState.query).toEqual('');
+  });
+
+  it('when a executeSearch fulfilled is received and the payload contains TriggerQuery objects, it updates `state.query`', () => {
```

Comment:
I'm seeing back-ticks around state.query, but these test descriptions do no render markdown.

I'd suggest starting the expression with a `#` to signify that "this is a code reference". e.g. `updates #state.query`